src/ViewHoldingsRecord.js

```diff
@@ -81,7 +81,14 @@ class ViewHoldingsRecord extends React.Component {
       path: 'holdings-storage/holdings/:{holdingsrecordid}',
       resourceShouldRefresh: false,
       accumulate: true,
-      tenant: '!{location.state.tenantTo}'
+      tenant: (_q, _p, _r, _l, props) => {
+        const {
+          stripes,
+          location,
+        } = props;
+
+        return location?.state?.tenantTo || stripes.okapi.tenant;
```

Comment:
`stripes-connect` already does this check
https://github.com/folio-org/stripes-connect/blob/master/OkapiResource.js#L48